internal/venice-consumer/src/main/java/com/linkedin/venice/pubsub/kafka/KafkaPubSubTopic.java

```diff
@@ -0,0 +1,52 @@
+package com.linkedin.venice.pubsub.kafka;
+
+import com.linkedin.venice.meta.Version;
+import com.linkedin.venice.pubsub.api.PubSubTopic;
+
+
+public class KafkaPubSubTopic implements PubSubTopic {
+  private final String name;
+  private final boolean isRealTimeTopic;
+  private final boolean isVersionTopic;
+  private final boolean isReprocessingTopic;
```

Comment:
Why do we need three different flags for topic checks? Can we use an enum instead?

Reply:
I think we can provide enum like `PubSubTopicType` of `VERSION_TOPIC`, `REALTIME_TOPIC`, `REPROCESSING_TOPIC`, we may also need `CHANGE_CAPTURE_TOPIC` in the future.